Makefile

```diff
@@ -161,6 +173,20 @@ endif
 push-name:
 	@echo "pushed: $(IMAGE):$(VERSION)"
 
+manifest: .manifest-$(MULTIARCH_IMAGE) manifest-name
+.manifest-$(MULTIARCH_IMAGE):
+	@docker manifest create $(MULTIARCH_IMAGE):$(VERSION) \
```

Comment:
I'm seeing an issue here where if I try to build/push the same tag repeatedly, I get the following error:

```
refusing to amend an existing manifest list with no --amend flag
```

Unfortunately, adding the `--amend` flag doesn't quite give me the behavior I'm looking for, since it adds to, rather than replaces, the manifest list (e.g. if on build 1 I do amd64 only, then on build 2 I do ppc64le only, my manifest after build 2 contains both).

Seems the only way to delete a manifest list is by manually deleting from `~/.docker/manifests` -- we may need to do that prior to creating them, but want to get others thoughts here before going down that path.

Reply:
@clnperez your thoughts on this one (possible workaround)?

Reply:
when you do the push, make sure you use the `--purge` flag to wipe out the old bits that are stored locally. hopefully that works for you!

Reply:
ah, that works and seems reasonable, thanks @clnperez!

Reply:
@skriss Do let me know if this addresses your concern

Reply:
> @skriss Do let me know if this addresses your concern

Noted.

Reply:
@Prajyot-Parab please add the `--purge` flag to the `docker manifest push` commands in the Makefile, thanks!

Reply:
@skriss Should I go ahead and do these changes ?
`@docker manifest push --purge $(MULTIARCH_IMAGE):$(VERSION)`
`docker manifest push --purge $(MULTIARCH_IMAGE):latest`